scripts/common.py

```diff
@@ -33,24 +33,28 @@ def execute(cmd, abort = True):
     return (status, stdout.rstrip('\n'), stderr.rstrip('\n'))
 
 
-# Subroutine for writing "pretty" XML; copied from http://effbot.org/zone/element-lib.htm#prettyprint
 def indent(elem, level=0):
-  i = "\n" + level*"  "
-  if len(elem):
-    if not elem.text or not elem.text.strip():
-      elem.text = i + "  "
-    if not elem.tail or not elem.tail.strip():
-      elem.tail = i
-    for elem in elem:
-      indent(elem, level+1)
-    if not elem.tail or not elem.tail.strip():
-      elem.tail = i
-  else:
-    if level and (not elem.tail or not elem.tail.strip()):
-      elem.tail = i
+    """Subroutine for writing "pretty" XML; copied from
+    http://effbot.org/zone/element-lib.htm#prettyprint"""
+    i = "\n" + level*"  "
+    if len(elem):
```

Comment:
I prefer `len(elem) > 0` for clarity and to show intent.

Reply:
This is copy&paste code from the URL in the docstring, please go ahead and make any changes you think are necessary or useful.